### PR TITLE
Aarch64

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hadim** [comment]
1.6.0 build for aarch64 didn't get uploaded. I also switched aarch64 and ppc64le build to azure. 

Hope it's ok.

**Comment by hadim** [comment]
Here is the build error: https://cloud.drone.io/conda-forge/conda-package-handling-feedstock/2/1/2

If someone can just relaunch it and it works then we can close this PR.

**Comment by hadim** [comment]
This PR is needed for https://github.com/conda-forge/miniforge/pull/2#issuecomment-554669064.

File: conda-forge.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 appveyor:
   secure: {BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1}
-provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
```
Comment (hmaarrfk) [inline conda-forge.yml]: did it not finish on drone?

Comment (hadim) [reply conda-forge.yml]: there was an error on drone: https://cloud.drone.io/conda-forge/conda-package-handling-feedstock/2/1/2

**Comment by hadim** [comment]
I need some help from a Windows expert here please:

```
 c:\users\vssadministrator\appdata\local\temp\pip-req-build-8fp1_7w5\src\conda_package_handling\archive_utils_c.c(6): fatal error C1083: Cannot open include file: 'archive.h': No such file or directory
```

**Comment by nehaljwani** [comment]
The latest build of **_libarchive_static_for_cph** is _broken_ on Windows:


```
$ tar -tvf _libarchive_static_for_cph-3.3.3-h48f6ca5_2.tar.bz2  | grep archive.h

$ tar -tvf _libarchive_static_for_cph-3.3.3-hfe4db18_0.tar.bz2 | grep archive.h
-rw-rw-rw- 0/0           52466 2018-09-03 18:54 Library/include/archive.h
```

**Comment by hadim** [comment]
Thanks @nehaljwani . Bug reported: https://github.com/conda-forge/_libarchive_static_for_cph-feedstock/issues/5

**Comment by hadim** [comment]
@nehaljwani : actually `h9696bfa_2` does not seem to be broken. I guess it's not selected because it has the same build number as the broken one?

**Comment by hadim** [comment]
Actually, I think I am wrong about `_libarchive_static_for_cph`. See https://github.com/conda-forge/_libarchive_static_for_cph-feedstock/issues/5#issuecomment-554718649

**Comment by hadim** [comment]
ping @conda-forge/conda-package-handling: please review and merge

**Comment by nehaljwani** [comment]
This is the failed build: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=94582&view=logs (which proceeded with packaging because the errorlevel wasn't checked in the recipe after the command for cmake build)

**Comment by hadim** [comment]
@nehaljwani Yes, I saw that. I am going to make a PR to make the build fails when it should.

**Comment by hadim** [comment]
Once https://github.com/conda-forge/_libarchive_static_for_cph-feedstock/pull/7 is merged we can get rid of the fixed dep for `_libarchive_static_for_cph`.

File: conda-forge.yml (hunk 1 above)
Comment (isuruf) [reply conda-forge.yml]: Can you revert this now?

**Comment by hadim** [comment]
@isuruf: drone is fine but one of the Travis jobs is failing now... I don't get why there are two jobs in the first place. I can still set `ppc64le` back to `azure` if necessary.

**Comment by hadim** [comment]
One of them is on https://travis-ci.com/ and the other one on https://travis-ci.org. No idea why they are both giving different results. I guess both services have different default configurations.

We should ignore this or I can switch back to azure.

**Pull request merged**